upgrade/1.0/Stage_0_Prerequisites.md

```diff
@@ -0,0 +1,106 @@
+This check will also be conducted in the 'prerequisites.sh' script listed below and will fail if the system is not running CSM-0.9.4 or CSM-0.9.5.
+
+## Stage 0.1 - Install latest docs RPM
+
+1. Install latest document RPM package:
+
+    * Internet Connected
+
+        ```bash
+        ncn-m001# cd /root/
+        ncn-m001# wget https://storage.googleapis.com/csm-release-public/shasta-1.5/docs-csm/docs-csm-latest.noarch.rpm
+        ncn-m001# rpm -Uvh docs-csm-latest.noarch.rpm
+        ```
+
+    * Air Gapped
+
+        ```bash
+        ncn-m001# cd /root/
+        ncn-m001# rpm -Uvh [PATH_TO_docs-csm-*.noarch.rpm]
+        ```
+
+## Stage 0.2 - Update `customizations.yaml`
+
+Perform these steps to update `customizations.yaml`:
+
+1. Extract `customizations.yaml` from the `site-init` secret:
+
+   ```bash
+   ncn-m001# cd /tmp
+   ncn-m001# kubectl -n loftsman get secret site-init -o jsonpath='{.data.customizations\.yaml}' | base64 -d - > customizations.yaml
+   ```
+
+2. Update `customizations.yaml`:
+
+   ```bash
+   ncn-m001# /usr/share/doc/csm/upgrade/1.0/scripts/upgrade/update-customizations.sh -i customizations.yaml
+   ```
+
+3. Update the `site-init` secret:
+
+   ```bash
+   ncn-m001# kubectl delete secret -n loftsman site-init
+   ncn-m001# kubectl create secret -n loftsman generic site-init --from-file=customizations.yaml
+   ```
+
+4. If using an external Git repository for managing customizations ([as recommended](../../install/prepare_site_init.md#version-control-site-init-files)),
+   clone a local working tree and commit appropriate changes to `customizations.yaml`,
+   e.g.:
+
+   ```bash
+   ncn-m001# git clone <URL> site-init
+   ncn-m001# cp /tmp/customizations.yaml site-init
+   ncn-m001# cd site-init
+   ncn-m001# git add customizations.yaml
+   ncn-m001# git commit -m 'Remove Gitea PVC configuration from customizations.yaml'
+   ncn-m001# git push
+   ```
+
+5. Return to original working directory:
+
+   ```bash
+   ncn-m001# cd -
+   ```
+
+## Stage 0.3 - Execute Prerequisites Check
+
+1. Run check script:
+
+    * Internet Connected
+
+        ```bash
+        ncn-m001# /usr/share/doc/csm/upgrade/1.0/scripts/upgrade/prerequisites.sh --csm-version [CSM_RELEASE] --endpoint [ENDPOINT]
+        ```
+
+        **NOTE** ENDPOINT is optional for internal use. It is pointing to internal arti by default
+
+    * Air Gapped
+
+        ```bash
+        ncn-m001# /usr/share/doc/csm/upgrade/1.0/scripts/upgrade/prerequisites.sh --csm-version [CSM_RELEASE] --tarball-file [PATH_TO_CSM_TARBALL_FILE]
+        ```
+
+**`IMPORTANT:`** If any errors are encountered then potential fixs should be displayed where the error occurred. `IF` the upgrade prequesites.sh script does not provide guidance, then open a support ticket for guidance before proceeding.
+
+2. The script also runs the goss tests on the initial stable node (typically `ncn-m001`) where the latest version of CSM has been installed. Make sure the goss test pass before continue.
```

Comment:
- This step should start with the action item. Change to something like:
"Ensure the goss tests pass before continuing. The script runs the goss tests on the initial stable node (typically `ncn-m001`) where the latest version of CSM has been installed." 

- Do we need to provide guidance on how to check if the tests pass?